### ISSUE TITLE
Search for tag

### ISSUE THREAD

**Issue by gavinr** [Kovah/LinkAce#309]
**Is your feature request related to a problem? Please describe.**
If I have a lot of tags, and I want to see the list of bookmarks associated with that tag, it's hard to find that page since the "all tags" page is paginated.

**Describe the solution you'd like**
Ability to search or filter on the "All Tags" page

**Comment by Kovah** [comment]
That's a good idea and could be applied to lists too.

**Comment by gavinr** [comment]
Thanks! I'm happy to support this via a PR if you do not have time.

**Comment by Kovah** [comment]
Yes please, if you have the time I would love to accept a pr on this. 💚

**Comment by gavinr** [comment]
Great, what would you like the UI to look like? Something like this?

![image](https://user-images.githubusercontent.com/209355/132541170-2faf6c7a-4fbd-477c-b80b-30ed22df5f61.png)

**Comment by Kovah** [comment]
I would place it to the left of the button on the same line, maybe without the "search" on the button to keep it small.
